Documentation/rkt-build-rkt.md

```diff
@@ -0,0 +1,107 @@
+# coreos.com/rkt/builder
+
+This container contains all build-time dependencies in order to build rkt. It
```

Comment:
For all lines? For all of this I mostly aligned to 80 characters for all paragraphs. Just want to check before doing them all.